docs/standard/parallel-programming/snippets/cs/detached1.cs

```diff
@@ -38,9 +38,9 @@ public static async Task Main()
 //     Running antecedent task 1...
 //     Launching attached child tasks...
 //     Finished launching detached child tasks...
+//     Executing continuation of Task 1
+//        Attached child task #1 running
+//        Attached child task #5 running
 //        Attached child task #2 running
 //        Attached child task #3 running
-//        Attached child task #1 running
 //        Attached child task #4 running
-//        Attached child task #5 running
-//     Executing continuation of Task 1
```

Comment:
The output order is non-deterministic. Each run can produce a different output.
That's why the comment before the output says that it produces "**similar**" output.
Instead of changing the order, which doesn't solve the issue, I'd emphasize the fact that the order is non-deterministic, possibly with technical reasoning about why this is happening.

Reply:
> @IEvangelist David Pine FTE
> 
> The comment by @Youssef1313 is correct. The order of tasks finishing is not guaranteed in this sample. This change isn't necessary, and the related issue can be closed.

Yes, the numbered tasks are non-deterministic. This fix was to re-order the "Executing continuation of Task 1" which is guaranteed before the numbered tasks.  It was incorrectly at the bottom before - which is wrong.

Reply:
Got it. Thanks.